app/models/match.rb

```diff
@@ -54,6 +54,16 @@ def status
     status_updates.last.status
   end
 
+  def status_class
```

Comment:
Hm, would having methods like `status_good?`, `status_bad?`, `status_pending?` be helpful? They would basically be delegators I guess. Something like:

```ruby
def status_good?
  return false unless status_update
  status_update.good?
end

private

def status_update
  status_updates.last
end
```